gpt4all-bindings/python/docs/gpt4all_python_embedding.md

```diff
@@ -22,13 +24,123 @@ The embedding model will automatically be downloaded if not installed.
     ```
     [0.034696947783231735, -0.07192722707986832, 0.06923297047615051, ...]
     ```
-### Speed of embedding generation
-The following table lists the generation speed for text document captured on an Intel i913900HX CPU with DDR5 5600 running with 8 threads under stable load.
 
-| Tokens          | 128  | 512  | 2048 | 8129 | 16,384 |
-| --------------- | ---- | ---- | ---- | ---- | ---- |
-| Wall time (s)   | .02  | .08  | .24  | .96  | 1.9  |
-| Tokens / Second | 6508 | 6431 | 8622 | 8509 | 8369 |
+You can also use the GPU to accelerate the embedding model:
+
+=== "GPU Example"
+    ```py
+    from gpt4all import Embed4All
+    text = 'The quick brown fox jumps over the lazy dog'
+    embedder = Embed4All(device='gpu')
+    output = embedder.embed(text)
+    print(output)
+    ```
+=== "Output"
+    ```
+    [0.034696947783231735, -0.07192722707986832, 0.06923297047615051, ...]
+    ```
+
+### Nomic Embed
```

Comment:
You should note what the default context size is here before discussing anything about chunking.

Reply:
it should also be specified per model.

I would actually just make the list of embedding models above a table, have a column which is the compiled context length and then note that by default a chunking strategy is applied.